src/components/application/application.view.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+const ApplicationView: React.FC = () => {
+    return(
+        <>
+            <div className="application">
+                <h3 className="application__header">Demographics</h3>
+                <form className="application__form">
+                <section className="first-last-section">
+                    <div className="application__first-name">
+                        <label className="application__label">First Name</label>
```

Comment:
Accessibility is something important to consider when building an application! I know it's a lot to get into with all these random acronyms like WCAG and ARIA, but it comes down to providing access for people who use their computer/phone differently. React's accessibility tips are the most clear: https://reactjs.org/docs/accessibility.html#accessible-forms

With a label, you just need to link it to the input ID it's talking about. This is done with a JSX prop `htmlFor`.
```
<label htmlFor="namedInput">Name:</label>
<input id="namedInput" type="text" name="name"/>
```